src/main/emme/toolbox/import/import_network.py

```diff
@@ -284,8 +284,7 @@ def execute(self):
                 ("ASPD",      ("@speed_adjusted",      "HWY_TWO_WAY", "EXTRA", "Adjusted link speed (miles/hr)")),
                 ("YR",        ("@year_open_traffic",   "HWY_TWO_WAY", "EXTRA", "The year the link opened to traffic")),
                 ("PROJ",      ("@project_code",        "HWY_TWO_WAY", "EXTRA", "Project number for use with hwyproj.xls")),
-                ("FC",        ("type",                 "TWO_WAY",     "STANDARD", "Roadway functional class")),
```

Comment:
this appears to be reverting Cundo's recent update to include FFC.

Reply:
Fixed. I have cleaned this repo by manually updating some conflict files.